bears/python/PyCheckerBear.py

```diff
@@ -0,0 +1,40 @@
+import shlex
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+        DistributionRequirement)
+
+
+@linter(executable='pychecker',
+        output_format='regex',
+        output_regex=r'(?P<filename>\w+\.py):(?P<line>\d+): '
+                     r'(?P<message>.*)')
+class PyCheckerBear:
+    """
+    Find bugs in your Python source code.
+    The code for each function, class, and method is checked for possible
+    problems. Checks for unused globals and locals(module or variable), unused
+    method arguments and using a variable before setting it or if you are
+    redefining a function/class/method in the same scope.
+    """
+    LANGUAGES = {'Python', 'Python 2', 'Python 3'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='pychecker')}
```

Comment:
Use `PipRequirement` instead of `DistributionRequirement` as the bear is meant to run over python files.